data.md

```diff
@@ -236,9 +236,9 @@ The corresponding `<op>Word` operations automatically perform the correct modulu
     rule W0 -Word W1 => chop( (W0 +Int pow256) -Int W1 ) requires W0 <Int W1
     rule W0 *Word W1 => chop( W0 *Int W1 )
     rule W0 /Word 0  => 0
-    rule W0 /Word W1 => chop( W0 /Int W1 ) requires W1 =/=K 0
+    rule W0 /Word W1 => chop( W0 /Int W1 ) requires W1 =/=Int 0
     rule W0 %Word 0  => 0
-    rule W0 %Word W1 => chop( W0 modInt W1 ) requires W1 =/=K 0
+    rule W0 %Word W1 => chop( W0 modInt W1 ) requires W1 =/=Int 0
```

Comment:
1. For the consistency, the other rules need to be changed as well, like:
```
rule rule W0 %Word W1 => 0 requires W1 ==Int 0
```
2. I remember that @MrChico systematically replaced this type of rules. Are these simply missed at that time? Or, did we decide not to change these rules? I don't remember, but just want to double-check.